packages/stream_chat_flutter/README.md

```
@@ -1,30 +1,52 @@
-- [Chat UI Kit](https://getstream.io/chat/ui-kit/)
-- [UI Docs](https://getstream.io/chat/docs/sdk/flutter/stream_chat_flutter/introduction/)
-- [Chat Client Docs](https://getstream.io/chat/docs/flutter-dart/?language=dart)
-
-**V4 Migration Guide**
-
-For upgrading from V3 to V4, please refer to the [V4 Migration Guide](https://getstream.io/chat/docs/sdk/flutter/guides/migration_guide_4_0/)
+- [Flutter Chat Tutorial](https://getstream.io/chat/flutter/tutorial/)
+- [UI Docs](https://getstream.io/chat/docs/sdk/flutter/stream-chat-flutter/introduction/)
+- [V10 Migration Guide](https://getstream.io/chat/docs/sdk/flutter/guides/migration-guide-10-0/)
 
 ### Changelog
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)